docs/snippets/modules/chains/additional/multi_retrieval_qa_router.mdx

```diff
@@ -35,7 +35,7 @@ retriever_infos = [
     },
     {
         "name": "pg essay", 
-        "description": "Good for answer quesitons about Paul Graham's essay on his career", 
+        "description": "Good for answer questions about Paul Graham's essay on his career", 
```

Comment:
guessing meant to be "answering", while we're at it?

Reply:
Good catch. Fixed with [c4d6dcb](https://github.com/hwchase17/langchain/pull/7416/commits/c4d6dcb22b7a27c4295026576684ba8600a364b1).